resolwe_bio/docker_images/rnaseq/Dockerfile

```diff
@@ -0,0 +1,83 @@
+FROM docker.io/resolwe/base:ubuntu-16.04
+
+LABEL MAINTAINER Resolwe Bioinformatics authors https://github.com/genialis/resolwe-bio
+
+RUN apt-get update && \
+    apt-get -y install --no-install-recommends software-properties-common && \
+
+    apt-get -y install --no-install-recommends \
+      python \
+      python-pip \
+      samtools \
+      unzip \
+      wget \
+      # Required for fastqc.
+      default-jre \
+      perl \
+      && \
+    
+    # Required for HTSeq.
+    echo "Installing numpy..." && \
+    pip install numpy==1.13.1 && \
```

Comment:
Why not install python packages using pip3? Please check if it works as desired.